### PR TITLE
Bump to core 0.11.0.

### PR DESCRIPTION
**Pull request by roznawsk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,8 +55,8 @@ defmodule Mixing.Pipeline do
       file_src: %Membrane.File.Source{location: "/tmp/input.wav"},
       parser: Membrane.WAV.Parser,
       converter: %Membrane.FFmpeg.SWResample.Converter{
-        input_caps: %Membrane.RawAudio{channels: 1, sample_rate: 16_000, sample_format: :s16le},
-        output_caps: %Membrane.RawAudio{channels: 2, sample_rate: 48_000, sample_format: :s16le}
+        input_stream_format: %Membrane.RawAudio{channels: 1, sample_rate: 16_000, sample_format: :s16le},
+        output_stream_format: %Membrane.RawAudio{channels: 2, sample_rate: 48_000, sample_format: :s16le}
```
Comment (pkrucz00) [inline README.md:59]: Should it be `*_stream_format` or `*_accepted_format`. I think it should be the latter

File: test/parser_test.exs [hunk 2]
```diff
@@ -12,9 +12,8 @@ defmodule Membrane.WAV.ParserTest do
   @input_path Path.expand("fixtures/input.wav", __DIR__)
   @reference_path Path.expand("fixtures/reference.raw", __DIR__)
 
-  defp perform_test(elements, links) do
-    pipeline_options = %Pipeline.Options{elements: elements, links: links}
-    assert {:ok, pid} = Pipeline.start_link(pipeline_options)
+  defp perform_test(structure) do
+    assert {:ok, _supervisor_pid, pid} = Pipeline.start_link(structure: structure)
```
Comment (pkrucz00) [inline test/parser_test.exs:16]: This is `Testing` so the return should be `pipeline` not a tuple

File: mix.exs [hunk 3]
```diff
@@ -36,13 +36,17 @@ defmodule Membrane.WAV.Plugin.Mixfile do
...
+      {:membrane_file_plugin, "~> 0.13.0", optional: true},
       {:ex_doc, "~> 0.26", only: :dev, runtime: false},
       {:dialyxir, "~> 1.1", only: :dev, runtime: false},
       {:credo, "~> 1.6", only: :dev, runtime: false},
-      {:membrane_ffmpeg_swresample_plugin, "~> 0.15.0", only: :test}
+      # {:membrane_ffmpeg_swresample_plugin, "~> 0.15.0", only: :test}
+      {:membrane_ffmpeg_swresample_plugin,
+       github: "membraneframework/membrane_ffmpeg_swresample_plugin",
+       branch: "core-v0.11",
+       only: :test}
```
Comment (pkrucz00) [inline mix.exs]: At the end of applying changes check if this branch is already merged

**Review by pkrucz00 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (roznawsk) [reply README.md:59]: It's `*_stream_format` in the latest release.

File: test/parser_test.exs (hunk 2 above)
Comment (roznawsk) [reply test/parser_test.exs:16]: According to documentation https://hexdocs.pm/membrane_core/Membrane.Testing.Pipeline.html#start_link/1 the return value is the same for Pipeline.start_link() and Testing.Pipeline.start_link().

**Review by pkrucz00 — APPROVED** [review]

**Pull request merged**